license-report.md

```diff
@@ -6270,12 +6270,12 @@ This report was generated on **Mon Apr 10 17:12:38 EEST 2023** using [Gradle-Lic
 
 The dependencies distributed under several licenses, are used according their commercial-use-friendly license.
 
-This report was generated on **Mon Apr 10 17:12:39 EEST 2023** using [Gradle-License-Report plugin](https://github.com/jk1/Gradle-License-Report) by Evgeny Naumenko, licensed under [Apache 2.0 License](https://github.com/jk1/Gradle-License-Report/blob/master/LICENSE).
+This report was generated on **Wed May 03 14:40:42 EEST 2023** using [Gradle-License-Report plugin](https://github.com/jk1/Gradle-License-Report) by Evgeny Naumenko, licensed under [Apache 2.0 License](https://github.com/jk1/Gradle-License-Report/blob/master/LICENSE).
 
 
 
 
-# Dependencies of `io.spine.protodata:protodata-protoc:0.8.0`
+# Dependencies of `io.spine.protodata:protodata-protoc:0.8.1`
```

Comment:
@dmdashenkov, I've tried using this version in #128, and it fails the Kotlin compilation because of duplicated declarations in the generated proto code.

Please try v0.8.1 in your PR and see if it works for you.

Reply:
Got some progress on this. Please have a look at [this commit](https://github.com/SpineEventEngine/ProtoData/pull/128/commits/5d5884db0a73b3cfbed3713e6e284efb31945c22) for the additional filtering that fixed the duplicated sources issue. It works with ProtoData v0.8.1 too.